File: crates/cli/tests/fixtures/javascript/block_comments.js
// expect: total=11 code=4 comment=5 blank=2 shebang=0
// jsdoc-style block comment fixture

/**
 * Add two numbers.
 */
function add(a, b) { return a + b; }

const x = add(3, 4);
const y = add(x, 1);
console.log(x + y);

File: crates/cli/tests/fixtures/rust/block_comments.rs
// expect: total=14 code=6 comment=5 blank=3 shebang=0
/* rust supports block comments */

/* multi-line
   block comment
   continues here */

fn add(a: i32, b: i32) -> i32 { a + b }

fn main() {
	let x = add(3, 4);
	let y = add(x, 5);
	println!("{}", y);
}
